### DISCUSSION TITLE
Ограничители

### DISCUSSION THREAD

**Discussion by gat3v** [Sokol-Games-GmbH/PixelCombats.PublicApi#235] [Ideas]
Считаю нужно добавить ограничители на то что бы не были **похожие и одинаковые** названия Режимов, Карт и так-же Списки режимов, Списки карт.

Можно сделать **Msg Info** и в ней же говорится о том что Уже есть такое название, Либо сделать добовление к одинаковому названию по типу этому _Примеру:_ Оригинал 'Peace', Созданное с таким же названием 'Peace_1'

Ну видно в примере что добовляется строка _ и количество одинаковых комнат (вроде так и есть)

**Comment by kkohno** [comment]
ну тут определитель нужно делать.. причем там будет то буква о то цифра 0 и пойди напиши алгоритм. ито обойдут))

**Comment by kkohno** [comment]
а режимы, карты списки - у них уникальный ID имя хоть тоже самое, главное ID. Однако есть такое, что назвать также могут режим... но на нем уже все свойства накопленные другие будут
